Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -229,7 +229,7 @@ bool CBoot::SetupWiiMemory(DiscIO::IVolume::ECountry country)
 
 		if (!settingsFileHandle.WriteBytes(gen.GetData(), SettingsHandler::SETTINGS_SIZE))
 		{
-			PanicAlertT("SetupWiiMemory: Cant create setting.txt file");
+			PanicAlertT("SetupWiiMemory: Can't create setting.txt file");
```

Comment:
I don't think it's a good idea to change something as insignificant as this and break the translations when we're trying to get 5.0 out :/